wagtail/admin/api/actions/unpublish.py

```diff
@@ -0,0 +1,39 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.core.actions.unpublish_page import UnpublishPageAction
+
+from .base import APIAction
+
+
+class UnpublishPageAPIActionSerializer(Serializer):
+    set_expired = fields.BooleanField(default=False, required=False)
```

Comment:
I think we should leave this out of the API, since it should only be set on pages that were scheduled to be unpublished and I'm not sure if we can validate that here.

Reply:
Alright!